### PR TITLE
Skeleton Fixes

### PR DESCRIPTION
**Pull request by Ghrehh**
## Description

Some changes to fix edge cases where skeleton loaders didn't show up on page load.

There are still some issues with Skeletons remounting as parent data providers finish their requests, which disrupts the skeleton shimmer animation somewhat. As far as I can tell this shouldn't be happening, but I don't have enough svelte/domain knowledge to figure out this one on my own.

I assume removing these guards is fine now because of the loading changes, it seems to work okay anyway. Otherwise I can change this to leverage the existing loading provider also. Any input welcome.

### CODE REVIEW

**Comment by aptkingston** [comment]
I'm a little bit apprehensive about removing all these loading guards just because they exist to prevent null references all over the place, but I guess the skeletons will still prevent the real components being rendered until loading is complete. There's bound to still be edge cases which cause errors after these changes but we can fix those whenever they happen.

I guess most stuff has already been written to always assume anything can be nullish since the majority of settings are fully user controllable.

If nothing is erroring with these changes then it's good to go I guess :+1:

**Comment by aptkingston** [comment]
Although these table block changes will cause major conflicts with the new side panels PR that I was about to merge, so if you wouldn't mind holding off until I've merged that one - then just take the new table block and remove the loading guards again?

**Comment by Ghrehh** [comment]
@aptkingston Could you review this when you get a moment please?

**Review by aptkingston — APPROVED** [review]
LGTM! The diffs are so awful - you'd think it would be able to detect things like just removing one layer of indendation from everything...

**Pull request merged**
